examples/mqtt/salediagnostics.json

```diff
@@ -1,8 +1,10 @@
 {
-   "eventTimestamp":"2020-04-22T10:30:03.548Z",
+   "eventTimestamp":"2020-04-28T13:49:50.814Z",
    "nfcReaderConnected":false,
    "printerConnected":true,
    "nodAvailable":true,
    "sapiAvailable":false,
-   "loggedIn":true
+   "loggedIn":true,
+   "journeyRef":"RUT:Line:440",

```

Comment:
We have now updated the schema for journey to include journeyId and journeyRef. This will be rolled out in production within a week and can be used by RuterSalg to populate the new field you need.

Reply:
Thank you, I have updated the pull request now to reflect your suggestions. Will also update the other pull request I have at https://github.com/RuterNo/ruterno.github.io/pull/11 .